py/__init__.py

```diff
@@ -122,6 +133,13 @@ class PathHolders:
                                               "protocol.codec.custom.endpoint_qualifier_codec")
     RaftGroupId = ImportPathHolder("RaftGroupId", "protocol")
     RaftGroupIdCodec = ImportPathHolder("RaftGroupIdCodec", "protocol.codec.custom.raft_group_id_codec")
+    SqlQueryId = ImportPathHolder("_SqlQueryId", "sql")
+    SqlQueryIdCodec = ImportPathHolder("SqlQueryIdCodec", "protocol.codec.custom.sql_query_id_codec")
+    SqlColumnMetadata = ImportPathHolder("SqlColumnMetadata", "sql")
+    SqlColumnMetadataCodec = ImportPathHolder("SqlColumnMetadataCodec", "protocol.codec.custom.sql_column_metadata_codec")
+    SqlError = ImportPathHolder("_SqlError", "sql")
+    SqlErrorCodec = ImportPathHolder("SqlErrorCodec", "protocol.codec.custom.sql_error_codec")
+    SqlPageCodec = ImportPathHolder("SqlPageCodec", "protocol.builtin")
```

Comment:
Why don't we have SqlPage import Path holder

Reply:
Because this is a built-in codec, not a custom one. We don't need `SqlPage` in this case

Reply:
I wanted to be sure we are on the same thing, I am talking about adding a seperate import path holder for SqlPage, not adding SqlPage inside SqlPageCodec import path holder

Reply:
It won't be used anywhere, so no point in adding it